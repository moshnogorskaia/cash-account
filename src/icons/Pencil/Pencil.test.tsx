import { Pencil } from "./Pencil";
import { render } from "@testing-library/react";
import "@testing-library/jest-dom";

describe("Pencil", () => {
    it("should render correctly", () => {
        const { asFragment } = render(<Pencil />);
        expect(asFragment()).toMatchSnapshot();
    });
});