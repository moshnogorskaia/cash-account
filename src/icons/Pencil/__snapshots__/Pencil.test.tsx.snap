// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pencil should render correctly 1`] = `
<DocumentFragment>
  <svg
    aria-hidden="true"
    height="24"
    viewBox="0 0 24 24"
    width="24"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      clip-rule="evenodd"
      d="M16.0608 2.24988C15.6631 2.24988 15.2817 2.40778 15.0005 2.68887L15.0003 2.68904L12.2197 5.46967L3.43904 14.2503L3.43875 14.2506C3.15798 14.5317 3.00018 14.9127 3 15.31V15.3103V19.5C3 19.8978 3.15804 20.2794 3.43934 20.5607C3.72065 20.842 4.10218 21 4.5 21H8.68969H8.69004C9.08733 20.9998 9.46832 20.842 9.74941 20.5613L9.74977 20.5609L18.4954 11.8131C18.5073 11.8026 18.5189 11.7917 18.5303 11.7803C18.5418 11.7689 18.5527 11.7572 18.5633 11.7452L21.311 8.99683L21.3111 8.99672C21.5922 8.71544 21.7501 8.33406 21.7501 7.93641C21.7501 7.53875 21.5922 7.15737 21.3111 6.87609L21.3108 6.87574L17.1211 2.68887C16.8398 2.40778 16.4584 2.24988 16.0608 2.24988ZM17.9989 10.1882L20.2501 7.93641L16.0608 3.74988L13.8107 6L17.9989 10.1882ZM12.75 7.06066L4.5 15.3107V19.5H8.68934L16.9383 11.249L12.75 7.06066Z"
      fill="currentColor"
      fill-rule="evenodd"
    />
  </svg>
</DocumentFragment>
`;
